### DISCUSSION TITLE
2024-04-18 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#3020] [Community Calls]
The next Modules Feedback Session will be on **Thursday, Apr 18, 2024 at 1500 UTC.**

See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

Please reply with any questions/topics you would like to see covered.

### Agenda

* [Modules Roadmap](https://github.com/orgs/cue-lang/projects/10)
* modules: support for .cueignore #2992

### Meeting details
https://meet.google.com/cvo-xdff-qhj

As usual, this session will be recorded for those unable to join us live.

### CUE Community shared calendar
If you want to stay up to date with future community calls, please [subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/O7TqE7m-aAE

### Slides/materials
https://docs.google.com/presentation/d/1UbgZPuvWjMO7N_fvZ13W_SbXvw7IxB_3YTWfPQy9zHA

### Auto-generated transcript
https://docs.google.com/document/d/1wkVcd7_rxKJ-y8cWOXOQR6FfVdhHO3YDr48vFc-jwOY

**Comment by myitcv** [comment]
Updated the agenda.

**Comment by dominikdm** [comment]
Added recording, deck and transcript.
